### DISCUSSION TITLE
匹配指定地区(colo 机场三字码) 如何返地区的ipv6

### DISCUSSION THREAD

**Discussion by lll423** [XIU2/CloudflareSpeedTest#363] [提问]
匹配指定地区(colo 机场三字码) 如何返地区的ipv6,默认好像只有ipv4

**Comment by XIU2** [comment]
"如何返地区的ipv6" ？没看懂。。。

**Reply by lll423** [reply]
./CloudflareST.exe -httping -cfcolo HKG   这样就是返回hkg地区的ip了，怎么样让它返回ipv6的ip进行测速

**Reply by XIU2** [reply]
软件默认参数 -f 指定的是 ip.txt 也就是 IPv4 数据，想要测速 IPv6，自然就需要手动指定为 ipv6.txt 呀( `-f ipv6.txt` )。
我在使用示例中写了： https://github.com/XIU2/CloudflareSpeedTest#-ipv4ipv6

**Reply by lll423** [reply]
能不能指定地区的ipv6 ?

**Reply by XIU2** [reply]
你前面不都写出来命令了吗？
另外，IPv6 因为数量太多，因此和 IPv4 不一样，软件并非是遍历所有 IP 段，而是随机生成一部分，即使如此数量依然不少，你即使指定地区 ( `-cfcolo HKG` )，也可能难以碰到指定地区，而且 IPv6 的节点地区相比 IPv4 要很少的多，因此完全看运气。

> 理论上，遍历 IPv6 所有段的话一般就会找到你要的地区，但那怕是要扫到下个世纪了。。。

我只提供目前 Cloudflare CDN 公开可用的 IPv6 段，想要具体某个地区的 IPv6，就自己碰运气或者想办法吧，我无法解决。
